src/ggrc/models/custom_attribute_value.py

```diff
@@ -213,6 +229,23 @@ def attribute_object(self):
     except:  # pylint: disable=bare-except
       return None
 
+  @property
+  def attribute_objects(self):
+    """Get all objects mapped to attributable object with given cav
+
+    Returns:
+        list of mapped objects to attributable object
+    """
+    cavs = self.attributable.custom_attribute_values
+    res = [cav.attribute_object for cav in cavs
```

Comment:
There is no particular need for `res` variable.
```suggestion
    return [cav.attribute_object for cav in cavs
```

Reply:
Done